<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Anton Kreuzkamp</author>
 <class>CharacterRunnerConfigUi</class>
 <widget class="QWidget" name="CharacterRunnerConfigUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>252</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="main">
     <property name="title">
      <string>Character Runner Config</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="layout_trigger">
        <item>
         <widget class="QLabel" name="label_trigger">
          <property name="text">
           <string>&amp;Trigger word:</string>
          </property>
          <property name="buddy">
           <cstring>edit_trigger</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="edit_trigger">
          <property name="showClearButton" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_shortcut">
        <item>
         <widget class="QLabel" name="label_alias">
          <property name="text">
           <string>Alias:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_alias">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_hex">
          <property name="text">
           <string>Hex Code:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_hex">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_list">
        <item>
         <widget class="QTreeWidget" name="list">
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>75</number>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>75</number>
          </attribute>
          <column>
           <property name="text">
            <string>Alias</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Code</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layout_Itemactions">
          <item>
           <widget class="QPushButton" name="addItem">
            <property name="text">
             <string>Add Item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteItem">
            <property name="text">
             <string>Delete Item</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edit_trigger</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
